pkg/proto/blobstore/blobstore.proto

```diff
@@ -159,3 +162,14 @@ message SizeDistinguishingeBlobAccessConfiguration {
     // Maximum size of blobs read from/written to the backend for small blobs.
     int64 cutoff_size_bytes = 3;
 }
+
+message GCSBlobAccessConfiguration {
+       // Name of the GCS bucket.
+       string bucket = 1;
+   
+       // Prefix for keys, e.g. 'bazel_cas/'.
+       string key_prefix = 2;
+
+       // Auth JWT location.
+       string auth = 3; 
```

Comment:
Pie in the sky: at some point I'd like to see if we could add support for dynamic load balancing/sharding/storage reconfiguration through a separate health checker service. To make this easier to use, more flexible, etc. I don't think it would make sense to source credentials and such from local files. Instead, we should simply place them in the configuration directly (e.g., using a `bytes` field).

If we still want the ability to split up the configuration across multiple files, we could look into something like [Jsonnet](https://jsonnet.org/).

Reply:
I actually started with that but as this is config file it was going to annoying to "stick" bytes into the config file

In general i go for the smallest change to achieve the goal.  So I am open to doing abit more here but would want a concrete answer for on disk how does this config file get loaded with the right bytes